oat-sa/extension-tao-delivery
# fix: TR-2756. Missing Arabic translations in browser tab title

https://oat-sa.atlassian.net/browse/TR-2756

How to test:
1. Sync your env with this PR https://github.com/oat-sa/tao-core/pull/3268
2. Run the test with arabic language.
3. Check that browser tab is translated.
